activation/activation_test.go

```diff
@@ -345,6 +345,9 @@ func TestBuilder_StartSmeshingAvoidsPanickingIfPrepareInitializerReturnsError(t
 		require.Fail(t, "test timed out or failed")
 	}
 
+	tab.stop()
+	tab.eg.Wait()
```

Comment:
I think we can just end the test here and create a new test.

first test checks if an error on `StartSmeshing` calls `log.Panic` and second test checks if an error on `PrepareInitializer` does not call `StartSession`.